### PR TITLE
Diagram to database

### PR DESCRIPTION
**Pull request by ab-noori**
### 🌟Features:

> - [ ] - The `patients`, `medical_histories`, `treatments`, `invoices`, and `invoice_items` relations are created.
> - [ ] - All the necessary `primary` and `foreign` keys are created.
> - [ ] - Finally the `many-to-many` relation table was also created.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  name VARCHAR(100) NOT NULL
+);
+
+-- Create medical_history_treatments to relate medical_history and treatments
+CREATE TABLE medical_history_treatments (
+  medical_history_id INT,
+  treatment_id INT,
+  PRIMARY KEY (medical_history_id, treatment_id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id),
+  FOREIGN KEY (treatment_id) REFERENCES treatments (id)
```
Comment (christianbiring1) [inline schema_based_on_diagram.sql:74]: - [x] Great work on creating the `medical_history_treatments` to make the `many-to-many` relationship possible :1st_place_medal:

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ab-noori @otmaneechchafyky , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 

## Highlights
- [ ] Great work on creating the respective database columns correctly as required in the projects requirements :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
- [ ] Kindly make sure you have `indexes` for all your `foreign keys` as specified in the project requirements.
Hint: `You can achieve that by saying: CREATE INDEX ON medical_histories (patient_id);`
`...`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (duskotmane) [reply schema_based_on_diagram.sql:74]: Thank you @christianbiring1

**Review by ab-noori — COMMENTED** [review]
**Hi @christianbiring1 👋,**
Thank you so much 🙏  for reviewing our project.
Your nice feedback is really appreciated 🙏.
I have checked the issue and made it fix 🎯.

thank-you-gif-22

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @ab-noori and @otmaneechchafyky     , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ab-noori — COMMENTED** [review]
**Hi @NahnahAJ  👋,**
Thank you so much 🙏  for reviewing and approving our project.
Your nice feedback is really appreciated 🙏.

Thank-You-Very-Much

**Pull request merged**